bazel_tools/haskell-cc-toolchain.patch

```diff
@@ -0,0 +1,238 @@
+diff --git a/haskell/CROSSTOOL.windows b/haskell/CROSSTOOL.windows
```

Comment:
Does it make sense to upstream this patch?

Reply:
Yes, I think so. I'll upstream it.